docs/framework_config.rst

```diff
@@ -21,8 +21,9 @@ Basic Usage
 .. code-block:: python
 
     from redbot.core import Config
+    from redbot.core import commands
```

Comment:
```suggestion
    from redbot.core import Config, commands
```

Reply:
This was intentionally left as two lines because shortly after there is an explanation of Config that goes line by line. Leaving it as two lines made the explanation easier to understand.

See: https://github.com/Cog-Creators/Red-DiscordBot/pull/4416/files#diff-249132a9932c4d5c2a61eceab315ecb1R50

For the remainder of the document where it is not explaining imports, the suggested format should be used.